File: LNPG_01_TIPOS_E_VARIAVEIS/C/calculadora_simples_1.c
#include <stdio.h>

int main() {
    float number_user_1_float;
    printf("Insira o primeiro numero:\n");
    scanf("%f", &number_user_1_float);
    
    float number_user_2_float;
    printf("Insira o segundo numero:\n");
    scanf("%f", &number_user_2_float);

    printf("%f + %f = %f\n", number_user_1_float, number_user_2_float, number_user_1_float + number_user_2_float);
    printf("%f - %f = %f\n", number_user_1_float, number_user_2_float, number_user_1_float - number_user_2_float);
    printf("%f * %f = %f\n", number_user_1_float, number_user_2_float, number_user_1_float * number_user_2_float);

    if (number_user_2_float == 0) {
        printf("Nao e possivel dividir por zero.\n");
    } 
    else {
        printf("%f / %f = %.2f\n", number_user_1_float, number_user_2_float, (float)number_user_1_float / number_user_2_float);
    }

    return 0;
}

File: LNPG_01_TIPOS_E_VARIAVEIS/Java/src/calculadora_simples_1.java
import java.util.Scanner;

public class calculadora_simples_1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Insira o primeiro numero:\n");
        double number_user_1_float = scanner.nextDouble();

        System.out.print("Insira o segundo numero:\n");
        double number_user_2_float = scanner.nextDouble();

        System.out.printf("%.2f + %.2f = %.2f\n", number_user_1_float, number_user_2_float, number_user_1_float + number_user_2_float);
        System.out.printf("%.2f - %.2f = %.2f\n", number_user_1_float, number_user_2_float, number_user_1_float - number_user_2_float);
        System.out.printf("%.2f * %.2f = %.2f\n", number_user_1_float, number_user_2_float, number_user_1_float * number_user_2_float);

        if (number_user_2_float == 0) {
            System.out.println("Nao existe divisao por 0.");
        } 
        else {
            System.out.printf("%.2f / %.2f = %.2f\n", number_user_1_float, number_user_2_float, number_user_1_float / number_user_2_float);
        }

        scanner.close();
    }
}
